package.json

```diff
@@ -69,7 +69,7 @@
     "cross-env": "^7.0.2",
     "delay-cli": "^1.1.0",
     "faucet": "0.0.1",
-    "linkinator": "^2.1.0",
+    "linkinator": "^2.0.6",
```

Comment:
@JoseJPR can you explain why you needed to roll back and lock this dependency version?

Reply:
Hi @zeke, sorry I think there was a problem in my last commit, I have pushed again with the correct version.